Makefile

```diff
@@ -0,0 +1,51 @@
+# Variables
+GENERATOR := openapi-generator 
+OUTPUT_DIR := .
+OPENAPI_FILE := api/openapi.yaml
+YQ := ./yq  # Use the local yq binary
```

Comment:
I would suggest running `yq` as a Docker container. A `Makefile` that automatically installs dependencies feels a bit risky to me. It could clash with other version of `yq` people may have in their laptops, etc.

You can see an example [here](https://github.com/bluelabs-eu/dragonara-outbox-feed/blob/main/.rc#L9).